code/modules/mob/living/carbon/emote.dm

```diff
@@ -77,10 +95,44 @@
 				'sound/creatures/monkey/monkey_screech_6.ogg',
 				'sound/creatures/monkey/monkey_screech_7.ogg')
 
+/datum/emote/living/carbon/snap
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = TRUE
+	restraint_check = TRUE
+	vary = TRUE
+
+/datum/emote/living/carbon/snap/can_run_emote(mob/user, status_check = TRUE, intentional)
+	if(!..())
+		return FALSE
+	// sorry pal, but you need an arm to snap
+	var/mob/living/carbon/C = user
+	return C.get_bodypart(BODY_ZONE_L_ARM) || C.get_bodypart(BODY_ZONE_R_ARM)
+
+/datum/emote/living/carbon/snap/one
+	key = "snap"
+	key_third_person = "snaps"
+	message = "snaps their fingers"
+	message_param = "snaps their fingers at %t"
+	sound = 'code/datums/emote_sounds/emotes/snap.ogg'
+
+/datum/emote/living/carbon/snap/two
+	key = "snap2"
+	key_third_person = "snaps twice"
+	message = "snaps their fingers twice"
+	message_param = "snaps their fingers at %t twice"
+	sound = 'code/datums/emote_sounds/emotes/snap2.ogg'
+
+/datum/emote/living/carbon/snap/three
+	key = "snap3"
+	key_third_person = "snaps thrice"
+	message = "snaps their fingers thrice"
+	message_param = "snaps their fingers at %t thrice"
+	sound = 'code/datums/emote_sounds/emotes/snap3.ogg'
```

Comment:
These don't really need to exist, I can't see them being used a lot

Reply:
I've personally played  and observed on the servers that have snap2 and snap3 and.. it gets used, it's not used as in ''haha fun snap3.ogg spam'' but more like... i've seen it used on situations where snap2 is used like ''go on GET TO MEDBAY NOW CHOP CHOP snap2'' or snap3 when someone makes snarky remarks and so on, for the regular spessman that doesn't try to roleplay, they won't give it use but having it available as another option for expressing something or to add it to follow a sentence feels nice

Reply:
Yeah these are useful imo